sdk/storage/azure-storage-file-share/azure/storage/fileshare/_shared/policies.py

```diff
@@ -82,7 +82,7 @@ def is_retry(response, mode):   # pylint: disable=too-many-return-statements
     """
     status = response.http_response.status_code
     if 300 <= status < 500:
-        # An exception occured, but in most cases it was expected. Examples could
+        # An exception occurred, but in most cases it was expected. Examples could
```

Comment:
Code in the `_shared` folder is supposed to be kept consistent between our four Storage packages (blob, files-datalake, files-share, and queue) so would you mind making these same changes in the other 3 packages?

Reply:
Hello @KdHyeon0661! Would you be open to co-authoring further improvements to `_shared` across the storage packages? I would love to do a pair programming session when you're available. Thank you and take care, my friend!

Reply:
I changes it with @wonhyeongseo.

Reply:
Ok, that is fine. Please just make sure the same exact changes that are here are applied to other `_shared` folders. Thanks.